cmd/package.go

```diff
@@ -0,0 +1,340 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/otiai10/copy"
+	"github.com/spf13/cobra"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+var install bool
+
+func init() {
+	rootCmd.AddCommand(packageCmd)
+	packageSnapCmd.Flags().BoolVarP(&install, "install", "i", false, "Install the snap after packaging it")
+	packageCmd.AddCommand(packageSnapCmd)
+	packageDebCmd.Flags().BoolVarP(&install, "install", "i", false, "Install the deb after packaging it")
+	packageCmd.AddCommand(packageDebCmd)
+}
+
+var packageCmd = &cobra.Command{
+	Use:   "package",
+	Short: "Package a desktop release",
+}
+
+var linuxPackageDependencies = []string{"libx11-6", "libxrandr2", "libxcursor1", "libxinerama1"}
+
+var packageSnapCmd = &cobra.Command{
+	Use:   "snap",
+	Short: "Package a desktop release as snap",
+	Run: func(cmd *cobra.Command, args []string) {
+		projectName := assertInFlutterProject().Name
+		assertHoverInitialized()
+
+		// Hardcode target to the current OS (no cross-compile support yet)
+		targetOS := runtime.GOOS
+
+		packageSnap(projectName, targetOS)
+	},
+}
+
+var packageDebCmd = &cobra.Command{
+	Use:   "deb",
+	Short: "Package a desktop release as deb",
+	Run: func(cmd *cobra.Command, args []string) {
+		projectName := assertInFlutterProject().Name
+		assertHoverInitialized()
+
+		// Hardcode target to the current OS (no cross-compile support yet)
+		targetOS := runtime.GOOS
+
+		packageDeb(projectName, targetOS)
+	},
+}
+
+func assertBuildOutputAvailable(projectName string, targetOS string) {
+	if _, err := os.Stat(outputBinaryPath(projectName, targetOS)); os.IsNotExist(err) {
+		fmt.Println("hover: No build outputs found\nhover: Please run `hover build` first")
+		os.Exit(1)
+	}
+}
+
+func packageSnap(projectName string, targetOS string) {
+	if targetOS != "linux" {
+		fmt.Println("hover: Snap only works on linux")
+		os.Exit(1)
+	}
+	assertBuildOutputAvailable(projectName, targetOS)
+	var snapcraftBin, err = exec.LookPath("snapcraft")
+	if err != nil {
+		fmt.Println("hover: Failed to lookup `snapcraft` executable. Please install snapcraft.\nhttps://tutorials.ubuntu.com/tutorial/create-your-first-snap#1")
+		os.Exit(1)
+	}
+	copyAsset("app/gitignore", filepath.Join("desktop", ".gitignore"))
+	snapDirectoryPath, err := filepath.Abs(filepath.Join("desktop", "snap"))
```

Comment:
Ths `desktop/snap` directory is only used for building the snap, is it not? It is always removed on each run of this command. Perhaps we could use a tmpdir instead, it's automatically cleaned up (in case something errors/panics in this function) and doesn't affect the user workspace.